app/routes/events/components/EventDetail/index.js

```diff
@@ -186,83 +193,58 @@ export default class EventDetail extends Component<Props> {
       'hours'
     );
 
-    const infoItems: Array<?{ key: string, value: Node }> = [
-      event.company && {
-        key: 'Arrangerende bedrift',
-        value: (
-          <Link to={`/companies/${event.company.id}`}>
-            {event.company.name}
-          </Link>
-        ),
-      },
-      event.createdBy && {
-        key: 'Forfatter',
-        value: (
-          <Link to={`/users/${event.createdBy.username}`}>
-            {event.createdBy.fullName}
-          </Link>
-        ),
-      },
-      event.responsibleGroup && {
-        key: 'Arrangør',
-        value: (
-          <span>
-            {event.responsibleGroup.name}{' '}
-            {event.responsibleGroup.contactEmail && (
-              <a href={`mailto:${event.responsibleGroup.contactEmail}`}>
-                {event.responsibleGroup.contactEmail}
-              </a>
-            )}
-          </span>
-        ),
-      },
-      {
-        key: 'Hva',
-        value: eventTypeToString(event.eventType),
-      },
-      {
-        key: 'Når',
-        value: <FromToTime from={event.startTime} to={event.endTime} />,
-      },
-      { key: 'Finner sted i', value: event.location },
+    const deadlines: Array<?{ key: string, value: Node }> = [
       event.activationTime
         ? {
             key: 'Påmelding åpner',
             value: <FormatTime time={eventRegistrationTime} />,
           }
         : null,
-      event.registrationDeadlineHours &&
-      !['OPEN', 'TBA'].includes(event.eventStatusType)
-        ? {
-            value: <FormatTime time={registrationCloseTime(event)} />,
-            key: 'Påmelding stenger',
-          }
-        : null,
-      event.unregistrationDeadlineHours &&
-      !['OPEN', 'TBA'].includes(event.eventStatusType)
-        ? {
-            value: <FormatTime time={unregistrationCloseTime(event)} />,
-            key: 'Avregistrering stenger',
-          }
-        : null,
       event.heedPenalties &&
       event.unregistrationDeadline &&
       !['OPEN', 'TBA'].includes(event.eventStatusType)
         ? {
-            key: 'Avregistreringsfrist',
+            key: 'Frist for prikk',
             value: <FormatTime time={event.unregistrationDeadline} />,
           }
         : null,
-    ];
-
-    const paidItems: Array<?{ key: string, value: Node }> = [
-      { key: 'Pris', value: `${event.priceMember / 100},-` },
       event.paymentDueDate
         ? {
             key: 'Betalingsfrist',
             value: <FormatTime time={event.paymentDueDate} />,
           }
         : null,
+      event.unregistrationDeadlineHours &&
+      !['OPEN', 'TBA'].includes(event.eventStatusType)
+        ? {
+            value: <FormatTime time={unregistrationCloseTime(event)} />,
+            key: 'Avregistrering stengt',
```

Comment:
I think "stenger" is better than "stengt", as we're talking about a time in the future

Reply:
When I think about it, it would be nice to remove the date once it has been closed, and just say "Avregistrering stengt", perhaps also with a Tooltip with the time it closed. That way people have to do less thinking 😅

Reply:
Yeah, I remember we talked about it. I'm torn between the fact that if the "Avregistrering stenger" is removed, people be stupid and dont understand why they can't un-register and mail the arrangør. But do you think people will see the tooltip?